plugin.php

```diff
@@ -17,9 +17,12 @@ function my_oxygen_elements_init()
         return;
     }
 
-    foreach ( glob(plugin_dir_path(__FILE__) . "elements/*.php" ) as $filename)
+    $path = plugin_dir_path(__FILE__) . "elements"
```

Comment:
You can keep using `glob` function just change the files pattern

```php
foreach ( glob(plugin_dir_path(__FILE__) . "elements/**/*.php" ) as $filename)
{
    include $filename;
}
```

Reply:
Does this also find files in the `"elements/*"` as well as `"elements/*/*/*...` directory?